ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -143,13 +143,18 @@ Item {
         }
 
         footer: WalletFooter {
+            id: footer
+
+            readonly property bool isHoldingSelected: !!walletStore.currentViewedCollectible && walletStore.currentViewedHoldingID !== ""
+            readonly property bool isCommunityCollectible: !!walletStore.currentViewedCollectible ? walletStore.currentViewedCollectible.communityId !== "" : false
+            readonly property bool isOwnerCommunityCollectible: !!walletStore.currentViewedCollectible ? (walletStore.currentViewedCollectible.communityPrivilegesLevel === Constants.TokenPrivilegesLevel.Owner) : false
```

Comment:
it can only be an OwnerCommunityCollectible if it's a CommunityCollectible, I'd add the previous property as a condition here.

Reply:
Updated! Thanks for the suggestion!